### PR TITLE
docs: synchronize startVitest API with docs

### LINKED ISSUE #3338: NodeAPI - `startVitest` parameter type differs from the example in the documentation
### Describe the bug

In the [example](https://vitest.dev/advanced/api.html#startvitest) from the documentation, the third [Options parameter](https://github.com/vitest-dev/vitest/blob/9dc6929380d510f9c6316d5e56b6d1d54f25fda7/packages/vitest/src/node/cli-api.ts#L26) is not passed. However, in actual usage, the TS type prompts that the third parameter is required.

### Reproduction

image
↓
image

### System Info

```shell
.
```

### Used Package Manager

npm

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide/).
- [X] Check that there isn't [already an issue](https://github.com/vitest-dev/vitest/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vitest-dev/vitest/discussions) or join our [Discord Chat Server](https://chat.vitest.dev).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### PR DESCRIPTION
**Pull request by sheremet-va**
Closes #3338

### CODE REVIEW

**Comment by sheremet-va** [comment]
@dammy001 can you help with review?

File: docs/advanced/api.md [hunk 1]
```diff
@@ -28,6 +28,12 @@ If watch mode is not enabled, Vitest will call `close` method.
 
 If watch mode is enabled and the terminal supports TTY, Vitest will register console shortcuts.
 
+You can pass down a list of filters as a second argument. Vitest will run only tests that contain at least one of passed down strings in their filepath.
+
+The third argument is CLI arguments that will override any other test config options.
+
+You can also pass down whole Vite config as the forth argument that will override any other user options.
```
Comment (dammy001) [inline docs/advanced/api.md:35]:
```suggestion
Alternatively, you can pass in the complete Vite config as the fourth argument, which will take precedence over any other user-defined options.
```

File: docs/advanced/api.md [hunk 2]
```diff
@@ -28,6 +28,12 @@ If watch mode is not enabled, Vitest will call `close` method.
 
 If watch mode is enabled and the terminal supports TTY, Vitest will register console shortcuts.
 
+You can pass down a list of filters as a second argument. Vitest will run only tests that contain at least one of passed down strings in their filepath.
+
+The third argument is CLI arguments that will override any other test config options.
```
Comment (dammy001) [inline docs/advanced/api.md:33]:
```suggestion
Additionally, you can use the third argument to pass in CLI arguments, which will override any test config options.
```

File: docs/advanced/api.md [hunk 3]
```diff
@@ -28,6 +28,12 @@ If watch mode is not enabled, Vitest will call `close` method.
 
 If watch mode is enabled and the terminal supports TTY, Vitest will register console shortcuts.
 
+You can pass down a list of filters as a second argument. Vitest will run only tests that contain at least one of passed down strings in their filepath.
```
Comment (dammy001) [inline docs/advanced/api.md:31]:
```suggestion
You can pass down a list of filters as a second argument. Vitest will run only tests that contain at least one of the passed-down strings in their file path.
```

**Comment by sheremet-va** [comment]
Thank you!

**Pull request merged**